.github/workflows/continuous_integration.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["pypy-3.9", "3.8", "3.9", "3.10", "3.11", "3.12-dev"]
+        python-version: ["pypy-3.9", "3.8", "3.9", "3.10", "3.11", "3.12", "3.13-dev"]
```

Comment:
Isn't 3.13 released now? Any reason to be using dev anymore?

Reply:
as of 2 weeks ago, I guess that's the case, yeah. I'll drop dev.